kie-server-parent/kie-server-tests/kie-server-integ-tests-jbpm/src/test/java/org/kie/server/integrationtests/jbpm/TimerIntegrationTest.java

```diff
@@ -91,4 +93,29 @@ public void testTimerStartEvent() throws Exception {
         assertTrue(distance2 < 10000);
 
     }
-}
+
+
+    @Test(timeout = 60 * 1000)
+    public void testTimerRoolbackTimerCancel() throws Exception {
+        String containerId = "timer-project-" + runtimeStrategy;
+        createContainer(containerId, releaseId, new KieServerConfigItem(KieServerConstants.PCFG_RUNTIME_STRATEGY, runtimeStrategy, String.class.getName()));
+
+        this.processClient.startProcess(containerId, "error-handling.test-rollback");
+        List<ProcessInstance> startedInstances = queryClient.findProcessInstancesByContainerId(containerId, null, 0, 10, null, false);
+
+        assertEquals(1, startedInstances.size());
+
+        // this should fail
+        try {
+            List<TaskSummary> summary = this.taskClient.findTasks(USER_JOHN, null, null);
```

Comment:
```suggestion
            List<TaskSummary> summary = this.taskClient.findTasks(USER_JOHN, 0, 10);
```

page null creates a 404 error:
` Error code: 404, message: RESTEASY003870: Unable to extract parameter from http request: javax.ws.rs.QueryParam("page") value is 'null' for public javax.ws.rs.core.Response org.kie.server.remote.rest.jbpm.RuntimeDataResource.getAllAuditTask(javax.ws.rs.core.HttpHeaders,java.lang.String,java.lang.Integer,java.lang.Integer,java.lang.String,boolean)`